src/components/ExecuteButton.js

```diff
@@ -0,0 +1,183 @@
+            Learn how to configure
+        </Button>
+    </TextContent>)
+})[status];
+
+const ExecuteButton = ({ isLoading, data, getConnectionStatus, remediationId, issueCount }) => {
+    const [ open, setOpen ] = useState(false);
+    const [ isUserEntitled, setIsUserEntitled ] = useState(false);
+    useEffect(() => {
+        window.insights.chrome.auth.getUser().then(user => setIsUserEntitled(user.entitlements.smart_management.is_entitled));
+    }, []);
+
+    const [ connected, disconnected ] = data.reduce(
+        ([ pass, fail ], e) => (e.connection_status === 'connected' ? [ [ ...pass, e ], fail ] : [ pass, [ ...fail, e ] ])
+        , [ [], [] ]
+    );
+
+    const rows = [ ...connected, ...disconnected ].map(con =>
+        ({ cells: [ con.executor_name || 'Direct connection', con.system_count, { title: styledConnectionStatus(con.connection_status) }]})
+    );
+    const connectedCount = connected.reduce((acc, e) => e.system_count + acc, 0);
+    const systemCount = data.reduce((acc, e) => e.system_count + acc, 0);
+
+    const pluralize = (number, str) => number > 1 ? `${number} ${str}s` : `${number} ${str}`;
+    return (isUserEntitled
+        ?  <React.Fragment>
+            <Button
+                onClick={ () => { setOpen(true); getConnectionStatus(remediationId); } }>
+        Execute playbook
+            </Button>
+            <Modal
+                className="ins-c-dialog"
+                width={ '50%' }
+                title={ 'Execute playbook' }
+                isOpen={ open }
+                onClose={ () => setOpen(false) }
+                isFooterLeftAligned
+                actions={ [
+                    <Button
+                        key="confirm"
+                        variant="primary"
+                        disabled={ connected.length > 0 }
+                        onClick={ () => setOpen(false) }>
+                        { `Execute Playbook on ${pluralize(connectedCount, 'system')}` }
+                    </Button>,
+                    <Button
+                        key="download"
+                        variant='link' onClick={ () => downloadPlaybook(remediationId) }>
+                        Download Playbook
+                    </Button>
+                ] }
+            >
+                <div>
+                    <TextContent>
+                        <Text component={ TextVariants.p }>
+                            { `Playbook contains ${pluralize(issueCount, 'issue')} affecting ${pluralize(systemCount, 'system')}.` }
+                        </Text>
+                        <Text component={ TextVariants.p }>
+                        Systems connected to a Satelite instance and configured with Receptor can be automatically remediated.
+                            To remediate other systems, download the Ansible playbook for this plan.
+
+                        </Text>
+                        <Text component={ TextVariants.h2 }>Connection status of systems</Text>
```

Comment:
I think this should be an `h3` or `h4` from looking at the mockups. It should be similar in size to the text above it, but bold.